core/schema/query.graphqls

```diff
@@ -1,3 +1,7 @@
+scalar Void
```

Comment:
I put these two new scalars here because they are probably useful outside of just modules/functions and thus didn't make sense to be defined in those schema files.